mux.go

```diff
@@ -260,6 +262,14 @@ func (r *Router) SkipClean(value bool) *Router {
 	return r
 }
 
+// UnescapeVars tells the router to escape the route variables before invoking
+// the handler. This is useful when used together with UseEncodedPath to avoid
```

Comment:
Can you add an example to the godoc here too, so that readers have a practical example to wrap their head around?

Reply:
Done at 7885b61.